lib/base.php

```diff
@@ -595,8 +596,13 @@ public static function init() {
 		// setup the basic server
 		self::$server = new \OC\Server(\OC::$WEBROOT, self::$config);
 		self::$server->boot();
+
 		$eventLogger = \OC::$server->getEventLogger();
 		$eventLogger->log('autoloader', 'Autoloader', $loaderStart, $loaderEnd);
+		$eventLogger->start('request', 'Full request after autoloading');
+		register_shutdown_function(function () use ($eventLogger) {
+			$eventLogger->end('request');
+		});
```

Comment:
```suggestion
		register_shutdown_function(fn () => $eventLogger->end('request'));
```
Arrow functions possible since https://github.com/nextcloud/server/pull/29286

Reply:
Not if we want to backport this

Reply:
Backporting would certainly be helpful